tests/legacy-cli/e2e/tests/jest/basic-version-29.ts

```diff
@@ -0,0 +1,13 @@
+import { applyJestBuilder } from '../../utils/jest';
+import { ng, silentNpm } from '../../utils/process';
+
+export default async function (): Promise<void> {
+  await applyJestBuilder();
+  await silentNpm('install', 'jest@29.5.0', 'jest-environment-jsdom@29.5.0', '--save-dev');
```

Comment:
Question: Do we need to continue to support v29? Since this is still experimental, I don't think we're on the hook for backwards compatibility yet and it's probably reasonable to limit this to the latest major unless we have reason to believe that will be exceptionally difficult for users to adopt.

Reply:
So it seems that this would need further investigation to get version 30 to work. Let's have a chat about this offline to see the next steps for this builder.